jetty-ee11/jetty-ee11-servlet/src/test/java/org/eclipse/jetty/ee11/servlet/MultiPartServletTest.java

```diff
@@ -214,7 +213,7 @@ public void testMaxRequestSize(boolean eager) throws Exception
         start(new HttpServlet()
         {
             @Override
-            protected void service(HttpServletRequest req, HttpServletResponse resp)
+            protected void service(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException
```

Comment:
All the changes in this test should be reverted, the old version tests the undeprecated/correct API.